core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.html

```diff
@@ -6,7 +6,7 @@
     </div>
   </div>
   <div class="row">
-    <div [ngClass]="{'col-sm-5': isLanguageRTL(), 'col-sm-3': !isLanguageRTL()}">
+    <div [ngClass]="{'col-sm-5': isLanguageRTL(), 'col-sm-5': !isLanguageRTL()}">
```

Comment:
This doesn't seem correct, we are assigning the same class regardless of the language being RTL or not? Ditto below.

Also please attach proof in RTL languages -- thanks!

Reply:
you are right @Vir-8 
I hadn't noticed that. I will remove the conditional rendering based on the isLanguageRTL() function. It's working fine with col-sm-5 in both cases. Here is the video proof for the same code.

https://github.com/user-attachments/assets/1d8ae6fe-4dd6-40c7-95a6-8ed5378093fd

Reply:
Thanks for checking, then let's set it to the appropriate col classes by default; regardless of language.